JavaScript/Topics/Data_Structures/README.md

```diff
@@ -1,295 +1 @@
 # Iterables, Sets and Maps in JavaScript objects
-
-## Maps:-
```

Comment:
Atleast explain something about Maps, Sets and Iterables in this README.md file
Then you can give the link to their respective files.

Reply:
Done sir.